package.json

```diff
@@ -6,6 +6,7 @@
   "main": "index.js",
   "scripts": {
     "build": "grunt build",
+    "optimize": "grunt optimize",
```

Comment:
Please forgive me. I always prefer to do this kind of change in a separate PR. I added it because I didn't want to install grunt globally to do a minified build. I snuck it into this PR because I just didn't have the will to open a separate PR.

Let me know if you want me to separate it.

Reply:
I'm ok with this, thanks for adding some context.